sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -106,6 +109,8 @@ public void setTransaction(final @NotNull String transaction) {
     final ITransaction tx = this.transaction;
     if (tx != null) {
       tx.setName(transaction);
```

Comment:
I would probably do `this.transactionName = transaction;` in this case too, just in case. I know this probably cannot happen, but if `tx` becomes null at some point later, this makes sure the name doesn't revert to the previous one.

Reply:
👍